server/graphql/v1/inputTypes.js

```diff
@@ -315,6 +328,7 @@ export const OrderInputType = new GraphQLInputObjectType({
     fromCollective: { type: CollectiveAttributesInputType },
     collective: { type: new GraphQLNonNull(CollectiveAttributesInputType) },
     tier: { type: TierInputType },
+    customFields: { type: new GraphQLList(CustomFieldsInputType) },
```

Comment:
This is simpler than that. This can just be an object with `key: value` pairs.

And we can name it `customValues` or `customData` to not mistake it with the `customFields` from the `Tier`.

Reply:
And this is where we can support `CustomFieldsInputType`.

Reply:
> This is simpler than that. This can just be an object with key: value pairs.

Is it not suppose to be an array of custom fields even though we have only one field currently, just against future expansion?

I.e 
```js
[
 {
  "name": "jsonUrl",
  "label": "URL of the JSON dependency file",
  "type": "url",
  "required": true,
  }
]
````

Reply:
In this case, in the Order table, we don't care about `label`, `type` and `required`, we just want keys and values.

```
{"jsonUrl": "http://backyourstack.com/heroku/dependencies.json"}
```